astropy/wcs/setup_package.py

```diff
@@ -286,13 +283,13 @@ def get_extensions():
     ]
 
     wcslib_config_paths = [
-        join(WCSROOT, "include", "astropy_wcs", "wcsconfig.h"),
-        join(WCSROOT, "include", "wcsconfig.h"),
+        str(WCSROOT / "include" / "astropy_wcs" / "wcsconfig.h"),
```

Comment:
Can you remind me what's the block on keeping these as Pathlib objects?

Reply:
`extension-helpers` 1.2.0 is needed. https://github.com/astropy/astropy/pull/17193 takes care of this part.

Reply:
Ok. So let's get this PR in, then.